packages/ui/src/council/hooks/useCouncilStatistics.ts

```diff
@@ -0,0 +1,22 @@
+import { map } from 'rxjs'
+
+import { useApi } from '@/common/hooks/useApi'
+import { useObservable } from '@/common/hooks/useObservable'
+
+export const useCouncilStatistics = () => {
+  const { api, connectionState } = useApi()
+
+  const councilSize = api?.consts.council.councilSize
+  const reward = api?.query.council.councilorReward().pipe(map((councilorReward) => councilSize?.mul(councilorReward)))
+
+  return {
+    budget: {
+      amount: useObservable(api?.query.council.budget(), [connectionState]),
+      refillPeriod: api?.consts.council.budgetRefillPeriod,
+    },
+    reward: {
+      amount: useObservable(reward, [connectionState]),
```

Comment:
Just to be sure that in the rather unlikely future we're safe with React hooks.

I would move hook usage from the return statement up and keep it together with other hooks. Right now there is a potential to introduce `if+return` somewhere before those hooks. This would in turn cause React to complain about hooks being out of order.